src/output.cpp

```diff
@@ -731,11 +731,22 @@ bool query_yn( const std::string &text )
            .option( "YES", allow_key )
            .option( "NO", allow_key )
            .cursor( 1 )
+           .allow_mouse_ctrl( mousectrl )
            .default_color( c_light_red )
            .query()
            .action == "YES";
 }
 
+bool query_yn_mouse( const std::string &text )
```

Comment:
Is there any reason to have this as special function instead of being the default?

In other words: why not enable mouse control all the time, and only disable it when required?

Reply:
See https://github.com/CleverRaven/Cataclysm-DDA/pull/58142#discussion_r889506071.

I think `query_yn` can have mouse control on by default because the mouse handling code is completely internal to the function. `query_popup` cannot though unless the mouse handling code is changed to be internal to `query_once`. Currently the code changes behavior on mouse clicks and some old code might not expect that.